File: examples/example-57/js/features/support/webdriver.js
const { Before, After } = require('@cucumber/cucumber');
const { Builder } = require('selenium-webdriver');

Before({ tags: '@webdriver' }, async function () {
    this.driver = await new Builder().forBrowser('firefox').build();
});

After({ tags: '@webdriver' }, async function () {
    if (this.driver) {
        await this.driver.quit();
    }
});

File: examples/example-64/ruby/features/support/webdriver.rb
require 'selenium-webdriver'

Before('@webdriver') { @driver = Selenium::WebDriver.for :firefox }

After('@webdriver') { @driver.quit }

Before('@screenplay') do
    Screengem::ActorMemory.instance.clear
    @actor = Class.new do
        include Screengem::Actor
        def name
            'Actor 1'
        end
    end.new
    @actor.remember(browser: Abilities::Browser.new(@driver, 'http://localhost:9999'))
end